plugins/configuration.rb

```diff
@@ -2,7 +2,7 @@ module Jekyll
   class ConfigurationBlock < Liquid::Block
     TYPE_LINKS = {
       'action'       => '/docs/scripts/',
-      'device_class' => '/docs/configuration/customizing-devices/#device-class',
+      'device_class' => '/integrations/homeassistant/#device-class',
       'template'     => '/docs/configuration/templating/',
       'icon'         => '/docs/configuration/customizing-devices/#icon',
```

Comment:
```suggestion
      'icon'         => /integrations/homeassistant/#icon',
```
This one Im not sure about, an "Icon" section could be added to the current page?